docs/source/pages/documentation.md

```diff
@@ -142,25 +141,22 @@ When the `all` argument is used for `--datatype` (`-dt`), the folders created de
 │   ├── ses-010_time-160248/
 │   │   ├── behav
 │   │   └── funcimg
-│   └── histology
+│   └── anat
```

Comment:
In all the example folder trees in this document, the `anat` is still shown on the subject level. Move to the session level, either together with the other datatypes, or in its own session.